### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by LeoNeek** [drknzz/GitHub-Achievements#95] [Q&A] [answered]
I'll mark it for you :)

**Comment by 0xpako** [comment, marked as answer]
I'll take that!

**Comment by mrdulin** [comment]
Hey, can we help each other?

**Reply by 0xpako** [reply]
surely! Take a look at #96

**Comment by ifarbod** [comment]
Can I take one here?

**Reply by 0xpako** [reply]
I'll make a thread for you in a second, keep an eye here please :)

**Reply by ifarbod** [reply]
Sure! Does it work with duplicates?

**Reply by 0xpako** [reply]
I believe so, check out #101

**Reply by ifarbod** [reply]
Good one, thanks!
